config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -67,6 +67,46 @@ periodics:
     testgrid-tab-name: build-master
     testgrid-alert-email: release-managers@kubernetes.io, release-team@kubernetes.io
 
+- interval: 1h
+  name: ci-kubernetes-build-canary
+  cluster: k8s-infra-prow-build
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/bootstrap:v20200924-7fcf543
+      args:
+      - --repo=k8s.io/kubernetes
+      - --repo=k8s.io/release
+      - --root=/go/src
+      - --timeout=240
+      - --scenario=kubernetes_build
+      - --
+      - --allow-dup
+      - --extra-version-markers=k8s-master
+      - --registry=gcr.io/kubernetes-ci-images
```

Comment:
hmm, what's this used for? if it's writing, we may want to setup and use another gcr repo so the build and build-canary jobs aren't racing / stomping overtop of each other's results?

Reply:
@justaugustus is it possible to use `k8s-staging-kubernetes` gcr repo ? Otherwise I would create another gcr repo.

Reply:
We'll want to k8s-release-dev here: https://github.com/kubernetes/k8s.io/pull/1110, https://github.com/kubernetes/k8s.io/issues/846#issuecomment-668795671

Reply:
The `k8s-release-dev` gcs bucket can't act as a gcr.io registry.  These buckets all live inside the k8s-release project.  Are you suggesting a gcr repo in that project?  I sortof feel like a staging gcr repo is appropriate here, but I don't have a strong opinion at the moment

Reply:
cs.k8s.io is down, I'm curious how/where these images are used

Reply:
@spiffxp You can use https://sourcegraph.com/cncf as backup : https://sourcegraph.com/search?q=repogroup:cncf+repo:github.com/kubernetes+%22gcr.io/kubernetes-ci-images%22&patternType=literal.

Reply:
Better still:
- https://sourcegraph.com/kubernetes
- https://sourcegraph.com/search?q=repogroup:kubernetes+%22gcr.io/kubernetes-ci-images%22&patternType=literal